src/components/Notes/Note.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div class="note" :class="{ expanded: isExpanded }" :style="noteStyle" @click="expandNote">
+    <div v-if="isExpanded" class="note-content">
+      <input v-model="noteText" placeholder="Add a note..." class="note-input" />
+      <img
+        src="@/assets/images/notes/arrow.svg"
+        alt="Arrow icon"
+        class="note-icon"
```

Comment:
Might want to change the naming for this to be more semantic, since we do already have a note icon in the menu dropdown.